ADAL/src/workplacejoin/mac/ADClientCertAuthHandler.m

```diff
@@ -38,36 +51,26 @@ + (void)resetHandler
 {
```

Comment:
true... *ponder*. Even worse, that means trying to tie it down to the timeout error of the connection. I'll have to poke around a bit to see if there's a good way to do that.